i18n/translations/en.ts

```diff
@@ -80,11 +80,6 @@ export const en = {
       explanation:
         "The secure inbox you own.\nConverse is built on XMTP, an open-source secure messaging protocol.",
     },
-    secondSignature: {
```

Comment:
can you help me understand where the need for this functionality was removed?

Reply:
It's not needed anymore from an SDK perspective see https://github.com/ephemeraHQ/converse-app/pull/1294#discussion_r1870040406